<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="StartScreen">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>152</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="GameScreen">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="drum1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="clock">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="drum1L">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum2L">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum3L">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>-2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="RestartButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="RestartScreen">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="final_score">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="score_word">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="GoButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ReturnButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>50</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="checkbg">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="RButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="CButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="test">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bad">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="good">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR CENA</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="people1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ReadyButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="intro1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="intro2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="introbg">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>introbg</zorder>
   <zorder>StartScreen</zorder>
   <zorder>ExitButton</zorder>
   <zorder>StartButton</zorder>
   <zorder>GameScreen</zorder>
   <zorder>drum2</zorder>
   <zorder>drum1</zorder>
   <zorder>clock</zorder>
   <zorder>drum1L</zorder>
   <zorder>drum2L</zorder>
   <zorder>drum3</zorder>
   <zorder>drum3L</zorder>
   <zorder>score</zorder>
   <zorder>RestartScreen</zorder>
   <zorder>RestartButton</zorder>
   <zorder>final_score</zorder>
   <zorder>score_word</zorder>
   <zorder>ExitButton2</zorder>
   <zorder>StopButton</zorder>
   <zorder>ReturnButton</zorder>
   <zorder>checkbg</zorder>
   <zorder>RButton</zorder>
   <zorder>CButton</zorder>
   <zorder>test</zorder>
   <zorder>bad</zorder>
   <zorder>good</zorder>
   <zorder>people1</zorder>
   <zorder>ReadyButton</zorder>
   <zorder>intro1</zorder>
   <zorder>intro2</zorder>
   <zorder>GoButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
